### PR TITLE
Add logo to README

### PR DESCRIPTION
**Pull request by anthumchris**

### CODE REVIEW

**Comment by surma** [comment]
Not opposed. @jakearchibald WDYT?

**Review by surma — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
+<img align="left" width="230px" src="https://squoosh.app/assets/icon-large.png" /><br>
+
 # [Squoosh]!
 
 [Squoosh] is an image compression web app that allows you to dive into the advanced options provided
 by various image compressors.
 
+<br><br><br>
```
Comment (jakearchibald) [inline README.md]: Is this to advance past the floated image? Feels a bit hacky.

Comment (AnthumChris) [reply README.md]: @jakearchibald Sadly, yes.  GitHub's markdown sanitizer doesn't leave us with many layout options.

**Comment by surma** [comment]
I can’t re-open this one as the original repository has been deleted.

**Pull request closed without merging**